docs/_jun_milestone_.md

```diff
@@ -0,0 +1,25 @@
+# Monthly Milestone - Jun 2020
+
+## Overview
+
+- Finish Q2 release
+
+## Goals
+
+- [ ] Clean up extension template and docs #5336
+- [ ] Failed to exclude properties by setting false in `fields` filter via url
+      #4992
+- [ ] Spike: robust handling of ObjectID type for MongoDB #3456
+- [ ] Integration with TypeORM #4853
+
```

Comment:
@agnes512, would it make sense to make a June goal to get the hasManyThrough original PR to `master`?